people.html

```diff
@@ -553,21 +553,6 @@ <h3>Alison Li (M.Sc. in Computer Science)</h3>
             </div>
         </div>
 
-        <h3>Jonathan Chan (M.Sc. in Computer Science)</h3>

```

Comment:
Add Jonathan to the list of SPL alumni at the bottom of this page?

Reply:
It's on L582!

Reply:
I MISSED it. I can't give off senior developer vibes this early aaaaaa